mlflow/tracking/_tracking_service/client.py

```diff
@@ -162,17 +162,25 @@ def get_experiment_by_name(self, name):
         """
         return self.store.get_experiment_by_name(name)
 
-    def create_experiment(self, name, artifact_location=None):
+    def create_experiment(self, name, artifact_location=None, experiment_tags=None):
         """Create an experiment.
 
         :param name: The experiment name. Must be unique.
         :param artifact_location: The location to store run artifacts.
                                   If not provided, the server picks an appropriate default.
+        :param experiment_tags: A dictionary of key-value pairs that are converted into
+                                  :py:class:`mlflow.entities.ExperimentTag` objects.
         :return: Integer ID of the created experiment.
         """
         _validate_experiment_name(name)
         _validate_experiment_artifact_location(artifact_location)
-        return self.store.create_experiment(name=name, artifact_location=artifact_location,)
+        experiment_tags = experiment_tags if experiment_tags else {}
+
+        return self.store.create_experiment(
+            name=name,
+            artifact_location=artifact_location,
+            experiment_tags=[ExperimentTag(key, value) for (key, value) in experiment_tags.items()],
+        )
```

Comment:
Do we have a test case for this?

Reply:
Added a unit test to `test_client.py` that ensures the mock store's `create_experiment` function was called with the `ExperimentTag` proto object